### DISCUSSION TITLE
How to implement api document versioning（api 文档的版本管理）

### DISCUSSION THREAD

**Discussion by maqingbo** [vuejs/vitepress#4037] [Q&A]
### 中文

在使用 vitepress 书写 api 文档时，由于多个版本之间的 api 内容不尽相同，如何实现对文档内容进行版本管理？

下面的 element-ui 文档截图可作为参考，点击不同版本号时可跳转不同的路由，请问 vitepress 能否实现类似的效果？

为实现类似效果，我曾尝试在服务器为每个版本新建个目录，每次发版时保存当前版本的文件不删除，但是这样只能保证高版本跳到低版本，因为低版本的打包文件里没有高版本的链接 🤔

### English

When using vitepress to write API documents, how to implement version management of document contents because the API contents between multiple versions are not exactly the same?

The following screenshot of the element-ui document can be used as a reference. Clicking different version numbers can jump to different routes. Can vitepress achieve a similar effect?

To achieve a similar effect, I have tried to create a new directory for each version on the server, and save the current version of the file without deleting it each time I release a version. However, this can only ensure that the higher version jumps to the lower version, because there is no link to the higher version in the packaged file of the lower version 🤔

### Images

![vitepress](https://github.com/user-attachments/assets/fd8565d6-b81a-465a-a1a1-94d7cd3afddf)

_vitepress ↑_

![element-ui new version](https://github.com/user-attachments/assets/d4e5a067-f1f9-45b5-a7b5-5c9f2e8d8cc3)

_element-ui new version ↑_

![element-ui old version](https://github.com/user-attachments/assets/0b71ac56-9e03-419e-a2fe-ad3683bbf561)

_element-ui old version ↑_

**Comment by brc-dd** [comment]
Versioning isn't currently supported. There are some third party plugins you may try. Please read - #109

**Reply by maqingbo** [reply]
> Versioning isn't currently supported. There are some third party plugins you may try. Please read - #109

Ok, thanks. Hope this function can be added in the future.
